.changeset/tame-moles-call.md

```diff
@@ -0,0 +1,8 @@
+---
+'@finos/legend-application-query': minor
+'@finos/legend-application-studio': minor
+'@finos/legend-graph': minor
+'@finos/legend-shared': minor
+---
+
+Allow re-arrange elements (class properties, tagged values, etc.) ([#303](https://github.com/finos/legend-studio/pull/303)).
```

Comment:
Let's split this changeset, the only package that should get the message and minor bump here is `@finos/legend-application-studio`

Reply:
--> `Allow re-arranging sub-elements (class properties, tagged values, etc.) in UML editors`